contracts/dao-dao-core/src/tests.rs

```diff
@@ -1150,6 +1150,52 @@ fn test_admin_permissions() {
     // DAO unpauses after 10 blocks
     app.update_block(|block| block.height += 11);
 
+    // Admin can directly pause the DAO
+    let res = app.execute_contract(
+        Addr::unchecked("admin"),
+        core_with_admin_addr.clone(),
+        &ExecuteMsg::Pause {
+            duration: Duration::Height(10),
+        },
+        &[],
+    );
+    res.unwrap();
+
+    let paused: PauseInfoResponse = app
+        .wrap()
+        .query_wasm_smart(core_with_admin_addr.clone(), &QueryMsg::PauseInfo {})
+        .unwrap();
+    assert_eq!(
+        paused,
+        PauseInfoResponse::Paused {
+            expiration: Expiration::AtHeight(start_height + 21)
+        }
+    );
+
+    // Admin can unpause the DAO
+    let res = app.execute_contract(
+        Addr::unchecked("admin"),
+        core_with_admin_addr.clone(),
+        &ExecuteMsg::Unpause {},
+        &[],
+    );
+    res.unwrap();
+
+    let paused: PauseInfoResponse = app
+        .wrap()
+        .query_wasm_smart(core_with_admin_addr.clone(), &QueryMsg::PauseInfo {})
+        .unwrap();
+    assert_eq!(paused, PauseInfoResponse::Unpaused {});
+
+    // DAO cannot unpause itself
+    let res = app.execute_contract(
+        core_with_admin_addr.clone(),
+        core_with_admin_addr.clone(),
+        &ExecuteMsg::Unpause {},
+        &[],
+    );
+    assert!(res.is_err());
```

Comment:
Would swap the order of these two tests and add one more:
- Check the DAO cannot unpause itself
- Check a random person can't pause it
- Check the admin can unpause successfully